GVFS/GVFS.Common/GVFSEnlistment.cs

```diff
@@ -90,7 +89,7 @@ public static GVFSEnlistment CreateWithoutRepoUrlFromDirectory(string directory,
                     return null;
                 }
 
-                return new GVFSEnlistment(enlistmentRoot, InvalidRepoUrl, gitBinRoot, gvfsHooksRoot);
+                return new GVFSEnlistment(enlistmentRoot, string.Empty, gitBinRoot, gvfsHooksRoot);

```

Comment:
@sanoursa we discussed sending null in here, however in the base if you send in null it will attempt to look up details for the repository [here](https://github.com/Microsoft/VFSForGit/blob/4a489012e5bef89adf2afd366b37928c5be2eec7/GVFS/GVFS.Common/Enlistment.cs#L30).  We don't want that in this case.  Previously there was a bug fix to diagnose explicitly saying don't fail if you can't get origin info.

We could look at other options like sending in a flag rather than the string.Empty.  Let me know what you think.

Reply:
I think this is fine. Any place where we try to use this as a URL, we will still fail. And your fix in DiagnoseVerb now ensures that we show a good display string.